package.json

```diff
@@ -16,12 +17,15 @@
     "@babel/runtime": "^7.6.0",
     "@hapi/hapi": "^18.4.0",
     "async": "^3.1.0",
-    "mysql": "^2.17.1"
+    "config": "^3.2.2",
```

Comment:
'Config' module is used if we want to change some settings based on `env`.

Documentation here: https://www.npmjs.com/package/config